smoke-tests/apps/TelemetryProcessors/src/smokeTest/resources/applicationinsights.json

```diff
@@ -78,8 +78,8 @@
         "type": "attribute",
         "actions": [
           {
-            "key": "http.url",
-            "pattern": "https?://[^/]+(?<httpPath>/[^?]*)",
+            "key": "url.path",
+            "pattern": "(?<httpPath>/[^?]*)",
```

Comment:
the equivalent of `http.url` in the new semconv is `url.full`

Reply:
this is a server span. url.full is only for client span

Reply:
oh yes, makes sense